addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,47 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
+    slide_ids = fields.One2many('slide.slide', 'survey_id', 'Certification')
+    slide_channel_ids = fields.One2many('slide.channel', compute='_compute_slide_channel_ids')
+    courses_count = fields.Integer("Linked Courses", compute="_compute_slide_channel_ids")
```

Comment:
Minor detail: either single quote or double quote for your compute method, but not both :)

(I usually prefer single quotes for technical stuff, such as here, and double quotes for sentences/strings/help attributes, but it's a personal preference).

Reply:
Fixed it